### DISCUSSION TITLE
トラックボール基盤の購入

### DISCUSSION THREAD

**Discussion by kaui126** [Yowkees/keyball#481] [Q&A]
遊舎工房でKeyball61を購入したのですが、製作途中でトラックボールの基盤を紛失してしまい作成ができない状態になってしまいました。
BOTTHのはんだ付け済トラックボール読み取り基板のページが在庫なしになっているのですが購入できますでしょうか？

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

トラックボール基板の購入についてご案内を進めさせていただきたいので、白銀ラボのウェブショップにありますお問い合わせフォームからご連絡いただけますでしょうか。
https://shirogane-lab.net/inquiry
